src/mpilib20_init_finalise.F90

```diff
@@ -63,8 +66,10 @@ end subroutine mpilib20_init
   !>                        with no restrictions.
   subroutine mpilib20_init_thread(mpi_env, required)
     use mpi_f08, only: MPI_COMM_WORLD, MPI_INIT_THREAD, MPI_COMM_RANK, MPI_ABORT, &
-                       MPI_COMM_DUP
-    
+                       MPI_COMM_DUP, &
```

Comment:
Nice catch, although  why only MPI_COMM_DUP on this line?

- [ ] Fix formatting